test-core-development.cfg

```diff
@@ -0,0 +1,10 @@
+[buildout]
+extends =
+    https://raw.githubusercontent.com/4teamwork/ftw-buildouts/master/test-package.cfg
+    https://raw.githubusercontent.com/4teamwork/ftw-buildouts/master/test-versions.cfg
```

Comment:
@jone yeah, I was about to suggest not to include the Plone version at first, but then noticed that we then would also miss Plone-specific hotfixes - which would be great to have in tests.

`test-eggs += hotfix-eggs` (pseudocode) sounds good to me, and `hotfix-eggs` should then be controlled by KGS (whether or not that's being used for tests, dev or production shouldn't make a difference).